cg/meta/external_data.py

```diff
@@ -0,0 +1,53 @@
+import logging
+from pathlib import Path
+
+from housekeeper.store import models as hk_models
+from cg.apps.housekeeper.hk import HousekeeperAPI
+from cg.constants import HK_FASTQ_TAGS
+from cg.constants.constants import CaseActions
+from cg.models.cg_config import CGConfig
+from cg.store import Store, models
+
+LOG = logging.getLogger(__name__)
+
+
+class ExternalDataHandler:
+    """Class for handling the integration of external data into the common workflow."""
+
+    def __init__(self, config: CGConfig):
+        self.hk_api: HousekeeperAPI = config.housekeeper_api
+        self.status_db: Store = config.status_db
+        self.external_data_dir = Path(config.external.external_data_dir)
+
+    def curate_external_folder(self):
```

Comment:
What is the curate part? Looks like it just iters and adds?

Reply:
It's because it's still a draft 😉